build.gradle

```diff
@@ -47,6 +47,9 @@ repositories {
 }
 
 dependencies {
+    // We should not include javax.mail in the dependencies as it is included with App Engine SDK and adding it will cause
+    // conflicts. However, in a very unlikely scenario that it is required, we should repackage it or use other workarounds.
+    // See https://github.com/TEAMMATES/teammates/pull/7525 for more info.
```

Comment:
I don't think it's necessary to leave this comment in the codebase. Adding it to your commit message body is sufficient.